product_docs/docs/pgd/5/cli/command_ref/pgd_create-proxy.mdx

```diff
@@ -31,13 +40,17 @@ pgd create-proxy [flags]
 ```text
       --group-name string   group name
   -h, --help                help for create-proxy
+      --proxy-mode string   proxy mode (default, read-only, any); proxy will route connections to -
```

Comment:
This option in both CLI cmd and sql interface is optional. So, hopefully, people will care about it only when they are trying to enable proxy for read-only connections.

Reply:
"follow" (the write leader) might have worked better.

Reply:
Hmm, not sure : ). "follow" in isolation may not look right. 

@freza @ddipak do you have any opinion or suggestion here. I don't mind changing it as we still have some time. But `default` is kind of also indicate it will work as it worked previously (which hopefully then we could tell that it means Write Leader).